deluge/ui/hostlist.py

```diff
@@ -22,7 +22,7 @@
 
 DEFAULT_HOST = '127.0.0.1'
 DEFAULT_PORT = 58846
-LOCALHOST = ('127.0.0.1', 'localhost')
+LOCALHOST = (('127.0.0.1', 'localhost'), ('::1', 'localhost'))
```

Comment:
This change would break the lookup for an ip or host here. The `::1` ip just need appended to the tuple so not a tuple of tuples

Reply:
Just change it to `('127.0.0.1', 'localhost', '::1')`

Reply:
oh wow, oops I was thinking of it was like connection pairs, thats localhost not localclient. ok that makes sense.